### ISSUE TITLE
openSUSE Leap 15.2 packages do not work

### ISSUE THREAD

**Issue by seife** [owncloud/core#37474]
### Steps to reproduce
1. install current openSUSE Leap 15.2 beta/RC
2. add owncloud production repo and install owncloud according to the documentation
3. try to log in

### Expected behaviour
Owncloud should work as expected and as it does on Leap 15.1

### Actual behaviour
```
This version of ownCloud is not compatible with PHP 7.4                                                                               
You are currently running PHP 7.4.6.                                                                                                  
```

### Server configuration
**Operating system**: openSUSE Leap 15.2 Beta

**Web server:** apache-2.4.41

**Database:** mariadb-10.4.12

**PHP version:** 7.4.6

**ownCloud version:** (see ownCloud admin page) 10.4.1

**Updated from an older ownCloud or fresh install:** updated

**Where did you install ownCloud from:** http://download.owncloud.org/download/repositories/stable/openSUSE_Leap_15.2

**Signing status (ownCloud 9.0 and above):**

this and the remaining questions do not really seem relevant to this problem.

**Comment by phil-davis** [comment]
ownCloud 10.5.0 is coming with support for PHP 7.4
e.g. https://github.com/owncloud/core/releases/tag/v10.5.0beta2 is in the process of being made available for testing.

**Comment by seife** [comment]
Good. As Leap 15.2 is scheduled to be released in 4 weeks (https://en.opensuse.org/openSUSE:Roadmap), openSUSE users would be in for a bad surprise after updating their systems ;-)
(I solved it for now be downgrading to php-7.2 from the 15.1 repos)

**Comment by altruizine** [comment]
I just went through that surprise. OpenSUSE Leap 15.2 was released yesterday, but Owncloud 10.5.0 has not been released. Hence, the currently released Owncloud packages available from http://download.owncloud.org/download/repositories/stable/openSUSE_Leap_15.2 are all broken.

**Comment by altruizine** [comment]
As expected, Owncloud's 10.5.0 release fixed this issue. The OpenSUSE Leap 15.2 packages now work as expected.
